### ISSUE TITLE
Warning "gulp-util@3.0.8: Use individual modules instead of a grab-bag"

### ISSUE THREAD

**Issue by edmorley** [kevva/gulp-decompress#12]
Hi!

If I do a `yarn add mozjpeg` I get the following warning during installation:
```
warning mozjpeg > bin-build > download > gulp-decompress > gulp-util@3.0.8: Use individual modules instead of a grab-bag
```

It seems that `gulp-util` (used by `gulp-decompress`) is now deprecated, and the package author wants people to use individual packages instead ([see here](https://github.com/gulpjs/gulp-util/blob/master/README.md)).

Looking at this repo I see the following being used:
* `gutil.File` - which the gulp-util readme recommends replacing with https://www.npmjs.com/package/vinyl
* `gutil.PluginError` - which the gulp-util readme recommends replacing with https://www.npmjs.com/package/plugin-error

**Comment by phil-r** [comment]
`gulp-util` is deprecated

see:
- https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
- https://github.com/gulpjs/gulp-util/issues/143

**Comment by DeveloperWil** [comment]
Do you know when the new gulp-util free version will be pushed up to npm?

**Comment by pieterbeulque** [comment]
This is already resolved in #13 and waiting for merge

**Comment by kevva** [comment]
Fixed in https://github.com/kevva/gulp-decompress/commit/d3fdff17810f757056503b0f81a8c06f3fa9dc60.
